### ISSUE TITLE
LiveSettingProperty is set False for error message appeared on leaving Font-Size edit field blank in 'Preferences' window

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#365]
### Steps to Reproduce

1. Launch Xamarin Workbook application.
2. Navigate to 'Tools' menu item present in menu bar and Expand it
3. Navigate to 'Preferences' menu item and select it
4. Navigate to 'general' tab item and select it
5. Navigate to Edit box and leave edit field blank
6. Observe Inspect property for Error message appeared

### Expected Behavior
LiveSettingProperty should be True for error message appeared in 'Preferences' window and Screen reader should read the error message

### Actual Behavior
LiveSettingProperty is set False for error message appeared in 'Preferences' window
**Narrator Behavior:** Narrator is not announcing Error message appeared on screen
**NVDA Behavior:** Narrator is not announcing Error message appeared on screen
**Note:** While giving Alphabetic values in edit box, warning or error is not being announced By Screen readers 

### User Impact
Users who rely on screen reader will not be able to know the error message appearing in 'Preferences' window

### Screenshots
![error_message_inspect](https://user-images.githubusercontent.com/22956130/40725243-88a7cb34-6440-11e8-8046-aba018f8c9d2.png)
![error_message_narrator](https://user-images.githubusercontent.com/22956130/40725245-8bb67aa0-6440-11e8-85f4-ef53dfb02e4e.png)
![error_message_nvda](https://user-images.githubusercontent.com/22956130/40725249-8e6010ae-6440-11e8-9356-086d31ce9ade.png)
![error_message_alphabet_narrator](https://user-images.githubusercontent.com/22956130/40725258-91eabe9a-6440-11e8-9259-53cd29fa1e88.png)
![error_message_alphabet_nvda](https://user-images.githubusercontent.com/22956130/40725262-9619ec2a-6440-11e8-92c5-188bb5cc89be.png)

### Environment
Windows :Windows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)
NVDA Version: 2018.1

### Log File

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.2.12
